global.json

```diff
@@ -5,6 +5,7 @@
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19264.13",
     "Microsoft.DotNet.Helix.Sdk": "2.0.0-beta.19264.13",
+    "FIX-85B6-MERGE-9C38-CONFLICT": "1.0.0",
```

Comment:
😆

Reply:
I am curious, does 85B6 9C38 mean anything? I thought maybe you were trying to link to the issue number somehow: 36846

Reply:
no, it's a random guid to avoide having a real package with that name.